py-polars/tests/unit/namespaces/test_datetime.py

```diff
@@ -374,7 +374,7 @@ def test_dst_offset_invalid() -> None:
         ("s", pl.Series(values=[1_577_836_800, 1_580_613_610], dtype=pl.Int64)),
         (
             "ms",
-            pl.Series(values=[1_577_836_800_000, 1_580_613_610_000], dtype=pl.Int64),
+            pl.Series(values=[1_577_836_800_000, 1_580_613_610_987], dtype=pl.Int64),
```

Comment:
the expected value was actually wrong here - the input is `'"2020-02-02 03:20:10.987654321"'`, so with unit 'ms', this should indeed have the `987` part